manifests/init.pp

```diff
@@ -15,6 +15,7 @@
   String $config_file_owner = $::fail2ban::params::config_file_owner,
   String $config_file_group = $::fail2ban::params::config_file_group,
   String $config_file_mode = $::fail2ban::params::config_file_mode,
+  String $config_file_before = $::fail2ban::params::before_file,
```

Comment:
Can you please use `String[1]`? This ensures people don't provide an empty string.